skietbaan_fe/src/components/ScoreCapture.js

```diff
@@ -99,37 +103,47 @@ export default class search extends Component {
 
   Validate() {
     let Valid = false;
-    if( parseInt(this.state.score) < 1 || this.state.score === null || this.state.score === "" || (this.state.score % 1) !== 0) 
-    {
+    if (parseInt(this.state.score) < 1 || this.state.score === null 
+    || this.state.score === "" || (this.state.score % 1) !== 0) {
```

Comment:
this doesn't look right, is score a string or int?

Reply:
ok removed it